nssp/delphi_nssp/pull.py

```diff
@@ -59,6 +65,7 @@ def pull_nssp_data(socrata_token: str):
         results.extend(page)
         offset += limit
     df_ervisits = pd.DataFrame.from_records(results)
+    create_backup_csv(df_ervisits, backup_dir, custom_run, logger=logger)
```

Comment:
I saw that there's no test for the actual create_backup_csv function for the last PR. can we add test for that in this PR?